scripts/authors.scala

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+exec scala "$0" "$@"
+!#
+
+/*
+ * Usage:
+ *    authors.scala tag1 tag2
+ *
+ * or if on non unixy os:
+ *    scala authors.scala tag1 tag2
+ *
+ * requires scala 2.11.x+ and command line git on path
```

Comment:
I thought it might be simpler to compile if there's no scala on the command line but on the other hand, if you don't need it a lot it might make sense to just keep it here.